### DISCUSSION TITLE
Decied the final theme

### DISCUSSION THREAD

**Discussion by samiba6** [202306-NEA-DZ-FEW/madlibs-fennecs#31] [Polls]
Well finally I could come up with few ideas (keeping cartoonlike theme idea) actually others are nested please help me decided a background and assets for the theme choose among these backgrounds: 
Background one :
![test](https://github.com/202306-NEA-DZ-FEW/madlibs-fennecs/assets/51706925/1bc54e64-0e32-420f-8f20-1c7bdcf1370e)
Background two:
![bg3](https://github.com/202306-NEA-DZ-FEW/madlibs-fennecs/assets/51706925/aeac7521-ef02-4ad7-ac71-ab8a24e78fd4)
Background three:
![bg2](https://github.com/202306-NEA-DZ-FEW/madlibs-fennecs/assets/51706925/1807f87c-4b95-48c2-ae72-f2e58c71d7bd)

and choose one of the assets (assents could be used as buttons too):

Assets set one :
![planets](https://github.com/202306-NEA-DZ-FEW/madlibs-fennecs/assets/51706925/abdb515f-b62f-4c51-a3a4-add649c12f25)

Assets set Two:
![planets2](https://github.com/202306-NEA-DZ-FEW/madlibs-fennecs/assets/51706925/a426f05d-d4f5-4df4-88c7-998a4bdf78e2)

**Comment by samiba6** [comment]
Well since the vote poll is one option only I choose assets set one since it match the story aka playful dancing planets

**Comment by dzc0d3r** [comment]
Background 2 or 3 
asset 1 for sure
Thank you @samiba6 for finding there assets: 💯

**Comment by meryem08** [comment]
I prefer background 3 and asset 1

**Comment by Khalil-NOUI** [comment]
BG 2 is Good,the assets set two
